build.sh

```diff
@@ -131,21 +131,21 @@ function create_unvars_c {
 
 echo Creating unvars.c
 cat <<END1 >motherboard/unvars.c
-/**************************************************************************************\\
-*                             Apple Lisa 2 Emulator                                    *
-*                                                                                      *
-*              The Lisa Emulator Project  $VERSION                    *
-*                  Copyright (C) ${COPYRIGHTYEAR} $AUTHOR                                   *
-*                            All Rights Reserved                                       *
-*                                                                                      *
-*                        Reset Global Variables .c file                                *
-*                                                                                      *
-*            This is a stub file - actual variables are in vars.h.                     *
-*        (this is autogenerated by the build script. do not hand edit.)                *
-\**************************************************************************************/
+# /**************************************************************************************\\
+# *                             Apple Lisa 2 Emulator                                    *
+# *                                                                                      *
+# *              The Lisa Emulator Project  $VERSION                                     *
+# *                  Copyright (C) ${COPYRIGHTYEAR} $AUTHOR                              *
+# *                            All Rights Reserved                                       *
+# *                                                                                      *
+# *                        Reset Global Variables .c file                                *
+# *                                                                                      *
+# *            This is a stub file - actual variables are in vars.h.                     *
+# *        (this is autogenerated by the build script. do not hand edit.)                *
+# \**************************************************************************************/
 
 #define IN_UNVARS_C 1
-// include all the includes we'll (might) need (and want)
+# include all the includes we'll (might) need (and want)
```

Comment:
Actually this is creating a .C file, though it's inside a bash shell script, therefore the comment should infact be // not #

Reply:
Ok; I didn't know that But the # instead didn't cause any problem during compilation. Is it normal ?